.github/workflows/codecov.yml

```diff
@@ -0,0 +1,46 @@
+name: codecov
+on:
+  push:
+    branches: [main, 0.4.x]
+  pull_request:
+jobs:
+  # Run code coverage using cargo-llvm-cov then upload to codecov.io
+  job_code_coverage:
+    name: llvm-cov
+    runs-on: ubuntu-latest
+    env:
+      # `MSRV` should match project `cargo-llvm-cov` value for `rust-version`
+      MSRV: 1.64
+      # set version outside of `Cargo.toml` to avoid `cargo-llvm-cov` dev-dependency
+      LLVM_COV_VERSION: 0.5.23
+      COVERAGE_FILE: ./coverage.lcov
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v3
+      - name: toolchain ${{ env.MSRV }} minimal
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: ${{ env.MSRV }}
+          override: true
+      - name: install cargo-llvm-cov
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: cargo-llvm-cov --version ${{ env.LLVM_COV_VERSION }}
```

Comment:
Why are we not using the pre-build binaries as recommended for GitHub Actions? https://github.com/taiki-e/cargo-llvm-cov#on-github-actions

Reply:
Added `- uses: taiki-e/install-action@cargo-llvm-cov`